debian/dh-package-notes.install

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/dh-exec
 dh_package_notes usr/bin
-generate-package-notes.py => usr/bin/generate-package-notes
+generate-package-notes => usr/bin/generate-package-notes
```

Comment:
The `=>` is for renaming, so this wouldn't work - just drop the shbang at the top and change this to `generate-package-notes usr/bin`
also please remove dh-exec from debian/control and the +x bit on debian/dh-package-notes.install